lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -128,6 +128,8 @@ def check_alt_backend(alt_backend):
         "matplotlib.backends.backend_{}".format(backend))
 
     ax.plot([0, 1], [2, 3])
+    if fig.canvas.toolbar:  # i.e toolbar2.
+        fig.canvas.toolbar.draw_rubberband(None, 1., 1, 2., 2)
```

Comment:
codecov claims this line isn't being tested - is that a CI issue or is the if statement never being executed anywhere?

Reply:
At least locally, I can confirm that this is indeed executed...

Reply:
This line is run for all backends, and we skip a few, so it seems to be referring to that.